updated server readme to reflect the gg/server-token-probs-4088 commit

ibehnam opened the pull request:
added explanation for the API's completion result which now includes `completion_probabilities`. Also added a JSON schema that shows the type/structure of `completion_probabilities`.

ibehnam committed:
updated server readme to reflect the gg/server-token-probs-4088 commit

added explanation for the API's completion result which now includes `completion_probabilities`. Also added a JSON schema that shows the type/structure of `completion_probabilities`.

ggerganov commented on examples/server/README.md:
````diff
@@ -174,35 +174,48 @@ node index.js
-    `tokens_evaluated`: Number of tokens evaluated in total from the prompt
-
-    `truncated`: Boolean indicating if the context size was exceeded during generation, i.e. the number of tokens provided in the prompt (`tokens_evaluated`) plus tokens generated (`tokens predicted`) exceeded the context size (`n_ctx`)
+```
+{'$defs': {'Probability': {'properties': {'prob': {'description': 'The probability of the token.',
+     'title': 'Prob',
+     'type': 'number'},
+    'tok_str': {'description': 'The string representation of the token.',
+     'title': 'Tok Str',
+     'type': 'string'}},
+   'required': ['prob', 'tok_str'],
+   'title': 'Probability',
+   'type': 'object'}},
+ 'properties': {'content': {'description': 'The content associated with the probabilities.',
+   'title': 'Content',
+   'type': 'string'},
+  'probs': {'description': 'A list of probabilities for each token.',
+   'items': {'$ref': '#/$defs/Probability'},
+   'title': 'Probs',
+   'type': 'array'}},
+ 'required': ['content', 'probs'],
+ 'title': 'Completion',
+ 'type': 'object'}
````
Try to format it better - there is a `json` snippet further down in the same file as an example

ibehnam replied:
@ggerganov Okay, I made the JSON schema of `completion_probabilities` easier to understand. Plz lemme know if you think it's ok.

ibehnam committed:
simplified the `completion_probabilities` JSON schema 

It's now easier to understand what the structure of `completion_probabilities` looks like.

ggerganov committed:
minor : fix trailing whitespace
